cyclonedx/model/bom.py

```diff
@@ -52,7 +52,7 @@
 from .vulnerability import Vulnerability
 
 if TYPE_CHECKING:
-    from packageurl import PackageURL  # type:ignore[import]
+    from packageurl import PackageURL
```

Comment:
therefore, we need to update 
https://github.com/CycloneDX/cyclonedx-python-lib/blob/main/pyproject.toml#L51

and not only the lock file